addons/website/static/tests/tours/website_style_edition.js

```diff
@@ -29,12 +45,54 @@ wTourUtils.registerWebsitePreviewTour("website_style_edition", {
     // a #t=... at the end then removes the old one.
     extra_trigger: 'iframe html:not(:has(link[href$="web.assets_frontend.min.css"]))',
     run: checkFontSize,
+}, {
+    content: "Open the color combinations area",
+    trigger: '.o_we_theme_presets_collapse we-toggler',
+}, {
+    content: "Open a color combination",
+    trigger: '.o_we_cc_preview_wrapper',
+}, {
+    content: "Edit the background color of that color combination",
+    trigger: '.o_we_theme_presets_collapse we-collapse .o_we_so_color_palette:eq(0)',
+}, {
+    content: "Choose a color",
+    trigger: `.o_we_color_btn[style*="background-color:${TARGET_BODY_BG_COLOR}"]`,
+}, {
+    content: "Check the body background color was properly adapted",
+    trigger: 'iframe body',
+    extra_trigger: `
+        .o_we_theme_presets_collapse we-collapse .o_we_so_color_palette:eq(0) .o_we_color_preview[style*="${TARGET_BODY_BG_COLOR}"],
+        .o_we_theme_presets_collapse we-collapse .o_we_so_color_palette:eq(0) .o_we_color_preview[style*="${TARGET_BODY_BG_COLOR_V2}"]
+    `,
+    run: checkBodyBgColor,
+}, {
+    content: "Edit the text color of that color combination",
+    trigger: '.o_we_theme_presets_collapse we-collapse .o_we_so_color_palette:eq(1)',
+}, {
+    content: "Choose a color",
+    trigger: `.o_we_color_btn[style*="background-color:${TARGET_BODY_COLOR}"]`,
+}, {
+    content: "Check the body color was properly adapted",
+    trigger: 'iframe body',
+    extra_trigger: `
+        .o_we_theme_presets_collapse we-collapse .o_we_so_color_palette:eq(1) .o_we_color_preview[style*="${TARGET_BODY_COLOR}"],
+        .o_we_theme_presets_collapse we-collapse .o_we_so_color_palette:eq(1) .o_we_color_preview[style*="${TARGET_BODY_COLOR_V2}"]
+    `,
+    run: checkBodyColor,
 },
 ...wTourUtils.clickOnSave(),
 {
     content: "Check the font size is still ok outside of edit mode",
     trigger: 'iframe body:not(.editor_enable) #wrapwrap',
     run: checkFontSize,
+}, {
+    content: "Check the body background color is still ok outside of edit mode",
+    trigger: 'iframe body:not(.editor_enable)',
```

Comment:
You can replace this (and the next one) by something else, maybe just `iframe body`.
`:not(.editor_enable)` is the responsibility of `clickOnSave()`, I am trying to get rid of all those occurences in the code, see https://github.com/odoo/odoo/pull/129317